### PR TITLE
dijkastra serialized version

### PR DESCRIPTION
**Pull request by roadmonster**
Tested working. Need to inject more testing cases.

### CODE REVIEW

File: SerializedVersion.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    { 
+        System.out.println("Vertex \t\t Distance from Source"); 
+        for (int i = 0; i < V; i++) 
+            System.out.println(i + " \t\t " + dist[i]); 
+    } 
+  
+    // Function that implements Dijkstra's single source shortest path 
+    // algorithm for a graph represented using adjacency matrix 
+    // representation 
+    void dijkstra(int graph[][], int src)
```
Comment (xxx0624) [inline SerializedVersion.java]: Pls change the signature of this method to be same as the parallelized one

File: SerializedVersion.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                if (!sptSet[v] && graph[u][v] != 0 && dist[u] != Integer.MAX_VALUE && dist[u] + graph[u][v] < dist[v]) 
+                    dist[v] = dist[u] + graph[u][v]; 
+        } 
+  
+        // print the constructed distance array 
+        printSolution(dist); 
+    } 
+  
+    // Driver method 
+    public static void main(String[] args)
```
Comment (xxx0624) [inline SerializedVersion.java:106]: By the way, pls use these small test data to check if this one is correct

File: SerializedVersion.java (hunk 1 above)
Comment (roadmonster) [reply SerializedVersion.java]: Done

File: SerializedVersion.java (hunk 2 above)
Comment (roadmonster) [reply SerializedVersion.java:106]: Done

**Pull request merged**